example/src/app/a-bit-more-complex-component/a-bit-more-complex.component.less

```diff
@@ -0,0 +1,5 @@
+.kek { color: red }
+
+:host span {
+  color: red;
+}
```

Comment:
Is this file really required? Let's keep the test as simple as possible. Plus I think we do not have a preprocessor configured.

Reply:
I removed it.